features/dump.feature

```diff
@@ -32,3 +36,15 @@ Feature: The dump command
       Util\.system! scp -C user@www\.example\.com:~\/dumps\/dump_for_download.dump .*?\/tmp\/aruba\/tmp\/staging.dump
       """
       And the output should contain "> Dumped the staging database to tmp/staging.dump"
+
+  When I run `geordi dump staging -d primary`
+    Then the output should contain "# Dumping the database of staging"
+      And the output should contain "> Connecting to www.example.com"
+      And the output should contain "Util.system! ssh, user@www.example.com, -t, cd /var/www/example.com/current && bash --login -c 'dumple staging --for_download'"
+      And the output should contain "> Downloading remote dump to tmp/staging.dump"
+      # Omitting the absolute path in this regex (.*)
+      And the output should match:
+      """
+      Util\.system! scp -C user@www\.example\.com:~\/dumps\/dump_for_download.dump .*?\/tmp\/aruba\/tmp\/staging.dump
+      """
+      And the output should contain "> Dumped the staging database to tmp/staging.dump"
```

Comment:
Please have this as a separate scenario. For one, it is a separate usage scenario :wink: and secondly, you accidentally got a wrong test green this way: L43 must contain "dumple staging primary" for the database selection, but the test is green because it sees the previous output of the first run.

Please also adapt the command output to reflect the different mode (include "primary"). You may omit steps that were already tested elsewhere and have no relevance for this very feature. Please also test the output when loading that dump (with `--load`). Suggestion:

```suggestion
Scenario: Creating a dump of one of multiple remote databases and loading it locally
  When I run `geordi dump staging --database primary --load`
    Then the output should contain "# Dumping the primary database of staging"
      And the output should contain "Util.system! ssh, user@www.example.com, -t, cd /var/www/example.com/current && bash --login -c 'dumple staging primary --for_download'"
      And the output should contain "> Dumped the primary staging database to tmp/staging.dump"
      
      # Loading the dump
      And the output should contain "Sourcing dump into the development db"
      And the output should contain "Your development database has now the data of staging (primary database)."
```

:information_source: you may tag scenarios with `@announce` (place that tag above `Scenario`) to see the output of the test run.

Reply:
Ah okay thanks! I am not very familiar with cucumber steps 🙂